manager/repository/userrepo/user.go

```diff
@@ -0,0 +1,25 @@
+package userrepo
+
+import (
+	"fmt"
+
+	"github.com/ormushq/ormus/manager/entity"
+)
+
+func (a StorageAdapter) DoesUserExistsByEmail(email string) (bool, error) {
```

Comment:
Move to `repository/scylladb/` and implement

Reply:
moved but not implemented yet